test-integration/ganache-pre-2.2/test/Tested.test.js

```diff
@@ -0,0 +1,31 @@
+const { shouldFail } = require('openzeppelin-test-helpers');
+
+const Tested = artifacts.require('Tested');
+
+contract('Tested', function (accounts) {
+  context('shouldFail.reverting.withMessage', async function () {
+    beforeEach(async function () {
+      this.contract = await Tested.new(0);
```

Comment:
What is this 0? `Tested` doesn't seem to have a constructor.

Reply:
Ouch that's a copy-n-paste mistake. Will remote it.